app/controllers/admin/merchants_controller.rb

```diff
@@ -16,6 +16,12 @@ def update
     edit_form_submission
   end
 
+  def create
+    @merchant = Merchant.new(merchant_params)
+    @merchant.save
```

Comment:
This definitely works, but is there a reason to use .new and .save instead of .create?